pkg/identityserver/store/gateway.go

```diff
@@ -52,8 +53,9 @@ type Gateway struct {
 	StatusPublic   bool `gorm:"not null"`
 	LocationPublic bool `gorm:"not null"`
 
-	ScheduleDownlinkLate   bool `gorm:"not null"`
-	EnforceDutyCycle       bool `gorm:"not null"`
+	ScheduleDownlinkLate   bool  `gorm:"not null"`
+	EnforceDutyCycle       bool  `gorm:"not null"`
+	ScheduleAnytimeDelay   int64 `gorm:"default:0"`
```

Comment:
- This can be unsigned, it should never be lower than 0
- This can be `not null` as well, we don't need to differentiate betwen `0` and `nil`

Reply:
intially added as `int64` because `time.Duration` is an int64. Can change

Reply:
how about the `not null`?

Reply:
I am unsure whether adding `not null` here would delete existing rows from the database during migrate. Perhaps this should not be a problem along with a default value.

Reply:
If you put `not null;default0` a migration should create the column and set the value in all rows to 0. If you feel more comfortable leaving it null, that's fine with me too.

Reply:
added not null